docs/source/examples/response/timelines/post/simple_bare_bones.json

```diff
@@ -152,13 +180,13 @@
               },
               "electricityTotal": {
                 "kwh": [0.0],
```

Comment:
@kellyatradiantlabs Shouldn't we change the `kwh` from 0.0 to the equivalent of 28.38 MBtu?

I looked it up on the Internet, but I always get confusing results because of `mBtu` or `MBtu`.

![image](https://github.com/RadiantLabs/modeling-api-docs/assets/10285345/f9c32b78-70b3-4163-8c41-e2c5eb0e8b54)